### DISCUSSION TITLE
Blazor Static Form Behavior

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4280] [General]
This an FYI for developers creating forms in static rendering:

https://github.com/dotnet/aspnetcore/issues/55808

**Comment by thabaum** [comment]
@sbwalker I wonder if you would use css `display: none;` and change the CSS for the user to display the form?

I do this with .controls to hide the controls on sites I dont want, then go to F12 and 
```
.controls {
display:block;
}
```
for example.

**Reply by sbwalker** [reply]
@thabaum that is exactly what I asked Microsoft in terms of what would they recommend.

**Reply by thabaum** [reply]
@sbwalker gotcha, I feel it should work as well, but from the reply I understand the "Blazor SSR" seems to be limited when trying to talk with other functions dynamically after the page loads.  The issue I believe is that the form code seems to get lost from memory so when you call the form from code behind logic it is not available.  I will have to play with this some over the next week see if I can come up with any workarounds as it is very interesting find and I appreciate the example given to test some ideas on.
